### PR TITLE
Firebase Methods: Accept conditions

### PR DESCRIPTION
**Pull request by LinnierGames**
# What's New
- UserService methods to mark `hasAccpetedConditions` to true for the current user

### CODE REVIEW

**Comment by LinnierGames** [comment]
[User Accepts Privacy Agreement](https://trello.com/c/hIt1zudd/115-user-accepts-privacy-agreement)

File: KifuSF/UniversalExtensions+Firebase.swift [hunk 1]
```diff
@@ -14,6 +14,29 @@ extension DatabaseReference {
...
+            .child("users")
+    }
+    
+    /**
+     `/users/:user_uid`
+     */
+    static func user(_ user: String) -> DatabaseReference {
+        return users()
+            .child(user)
+    }
```
Comment (LinnierGames) [inline KifuSF/UniversalExtensions+Firebase.swift:41]:
```suggestion
    static func user(_ user: String) -> DatabaseReference {
        return rootDirectory
            .child(Subtrees.users)
               .child(user)
    }
```

Comment (LinnierGames) [reply KifuSF/UniversalExtensions+Firebase.swift:41]: Add an `enum` [here](https://github.com/Kifuapp/KifuSF/pull/19/files#diff-0f2ba8181ace2c1f68e27fa07b7f613cR13)

**Review by pondorasti — APPROVED** [review]
Second great PR! 👍

Let's hope the third one is just as good 😉

**Comment by LinnierGames** [comment]
🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥

**Pull request merged**